docs/source/routing/graphos-reporting.mdx

```diff
@@ -15,25 +15,29 @@ export APOLLO_KEY=<YOUR_GRAPH_API_KEY>
 export APOLLO_GRAPH_REF=<YOUR_GRAPH_ID>@<VARIANT_NAME>
 ```
 
+<a id="usage-reporting-via-opentelemetry-protocol-otlp"></a>
```

Comment:
@shorgi I want to keep old URL references working like this but it doesn't seem to work in the preview 😢  Do we have a blessed way to do this? eg this should refer to the renamed "GraphOS tracing ..." heading. https://www.apollographql.com/docs/deploy-preview/5ce69b97f28c4082b7e75e31/graphos/routing/graphos-reporting#usage-reporting-via-opentelemetry-protocol-otlp